src/towncrier/_project.py

```diff
@@ -16,26 +16,24 @@
 
 def _get_package(package_dir, package):
 
-    # Step 1: Try the dumbest and simplest thing that could possibly work.
-    # Yes, that means importing it. Call the cops, I don't care.
-
-    sys.path.insert(1, package_dir)
-
     try:
         module = import_module(package)
-    except ImportError as e:
-        print("Tried to import {}, but ran into this error: {}".format(package, e))
-        # wups that didn't work
-        module = None
-
-    # Don't leave trash in sys.path
-    sys.path.pop(0)
-
-    # Step 2: uhhhhhhh
-    # TBA
-
-    if not module:
-        raise Exception("Can't find your project :(")
+    except ImportError:
+        # Step 1: Try the dumbest and simplest thing that could possibly work.
+        # Yes, that means importing it. Call the cops, I don't care.
```

Comment:
I've kept this comment as is, but might make sense to rephrase - there was never a step 1 for example.

Reply:
Done in a fixup (please squash-merge): it also fixes the sys.path.pop.